package.json

```diff
@@ -8,7 +8,7 @@
     "Nicholas Hwang <nhwang@hubspot.com>"
   ],
   "engines": {
-    "node": ">=6.9"
+    "node": ">=6.2.2"
```

Comment:
I moved it down to `v6`. I had set it to `v6.2.2` because it was the lowest version I required. It should be more permissive now.